src/System.Management.Automation/security/nativeMethods.cs

```diff
@@ -1081,36 +1081,36 @@ internal static DWORD DestroyWintrustDataStruct(WINTRUST_DATA wtd)
         [StructLayout(LayoutKind.Sequential)]
         internal struct CRYPT_PROVIDER_CERT
         {
-            private DWORD _cbStruct;
+            private readonly DWORD _cbStruct;
             internal IntPtr pCert; // PCCERT_CONTEXT
-            private BOOL _fCommercial;
-            private BOOL _fTrustedRoot;
-            private BOOL _fSelfSigned;
-            private BOOL _fTestCert;
-            private DWORD _dwRevokedReason;
-            private DWORD _dwConfidence;
-            private DWORD _dwError;
-            private IntPtr _pTrustListContext; // CTL_CONTEXT*
-            private BOOL _fTrustListSignerCert;
-            private IntPtr _pCtlContext; // PCCTL_CONTEXT
-            private DWORD _dwCtlError;
-            private BOOL _fIsCyclic;
-            private IntPtr _pChainElement; // PCERT_CHAIN_ELEMENT
+            private readonly BOOL _fCommercial;
+            private readonly BOOL _fTrustedRoot;
+            private readonly BOOL _fSelfSigned;
+            private readonly BOOL _fTestCert;
+            private readonly DWORD _dwRevokedReason;
+            private readonly DWORD _dwConfidence;
+            private readonly DWORD _dwError;
+            private readonly IntPtr _pTrustListContext; // CTL_CONTEXT*
+            private readonly BOOL _fTrustListSignerCert;
+            private readonly IntPtr _pCtlContext; // PCCTL_CONTEXT
+            private readonly DWORD _dwCtlError;
+            private readonly BOOL _fIsCyclic;
+            private readonly IntPtr _pChainElement; // PCERT_CHAIN_ELEMENT
```

Comment:
We don't provide a constructor for these and they're readonly. How would these be set?

Reply:
Same question for the rest of the structs in this file actually.

Reply:
I'd ask why they private :-)

Looks like false positives.

Reply:
Yeah both are good questions here 😁 

Not sure how this struct is actually used to be honest, but I don't see how it would be useful at the moment.

Reply:
In fact, all these structures are created in P/Invoke, they are really readonly for C# and "private" here is "named unused".